lib/app.tsx

```diff
@@ -76,6 +76,12 @@ class AppComponent extends Component<Props> {
 
   handleShortcut = (event: KeyboardEvent) => {
     const { hotkeysEnabled } = this.props;
+
+    // Handle search shortcuts even if keyboard shortcuts are disabled.
+    if (!window.electron || !isMac) {
```

Comment:
Hah - I was going to comment that this conditional is weird and bad, and then I realized it's a copy of the one I wrote, and it's like this because I never figured out why the shortcuts don't fire on Win/Linux. Nice.

Reply:
Should we maybe change these to `if ( !(window.electron && isMac) )` instead to make it easier to read?

Reply:
I'd rather leave it as-is; ultimately I want to debug why they don't get fired on Win/Linux but it has been knocked off my list by more urgent work. I suppose since we now have it in two places we could conceivably create a new variable, like `shouldHandleBrowserShortcuts = !window.electron || !isMac`. I leave that to your discretion.